resources/assets/components/FilterBar/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+import DropdownFilter from '../DropdownFilter';
+import MultiValueFilter from '../MultiValueFilter';
+
+class FilterBar extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      filters: {
+        status: 'accepted',
+        tags: {
+          'good-photo': {
+            label: "Good Photo",
+            active: false,
+          },
+          'good-quote': {
+            label: "Good Quote",
+            active: false,
+          },
+          'hide-in-gallery': {
+             label: "Hide In Gallery 👻",
+             active: false,
+          },
+          'good-for-sponsor': {
+            label: "Good For Sponsor",
+            active: false,
+          },
+          'good-for-storytelling': {
+            label: "Good For Storytelling",
+            active: false,
+          },
+        }
+      }
+    };
+
+    // The component provides a function the children components can use to update the state.
+    this.updateFilters = this.updateFilters.bind(this);
+  }
+
+  updateFilters(value){
+    if (['pending', 'accepted', 'rejected'].includes(value)) {
+      this.setState((previousState) => {
+        const newState = {...previousState};
+        newState.filters.status = value;
+        return newState;
+      });
+    }
+
+    if (['good-photo', 'good-quote', 'hide-in-gallery', 'good-for-sponsor', 'good-for-storytelling'].includes(Object.keys(value)[0])) {
+      let key = Object.keys(value)[0];
+
+      this.setState((previousState) => {
+        const newState = {...previousState};
+        newState.filters.tags[key].active = Object.values(value)[0];
+        return newState;
+      });
```

Comment:
@sbsmith86 everything is working except for this one thing! On the first page load, when I click a tag, it doesn't show up as active but the state updates correctly (saw this when I `console.log`ed the state out). However, when I click `FILTER`, the tag is highlighted as expected. When I click on another tag, it becomes active on the click. This functionality only seems to work after you click `FILTER` and I'm not sure why? Here's a gif to show what's going on: 
![sep-18-2017 11-10-28](https://user-images.githubusercontent.com/9019452/30549159-10c58940-9c62-11e7-8f25-78ac07acb14c.gif)